soundata/annotations.py

```diff
@@ -2,6 +2,28 @@
 """
 import numpy as np
 
+#: Time units
+TIME_UNITS = {
+    "seconds": "seconds",
+    "miliseconds": "miliseconds",
+}
+
+#: Label units
+LABEL_UNITS = {"open": "no strict schema or units"}
+
+#: Azimuth units
+AZIMUTH_UNITS = {
+    "radians": "values in the interval [-2*pi, 2*pi]",
+    "degrees": "values in the interval [-360, 360]",
```

Comment:
n00b question: why do we need the negative range? Doesn't [0, 360] cover everything?

Reply:
same for radians?

Reply:
I would think so, but I'm trying to be as general as possible. Basically accounting for datasets that might annotate location with negative angles e.g. if the "sight of view" is around 60 degrees with respect to the center of the array, the sound source might be annotated moving from -30 to 30 and so on. I think that's how it is done in `pyroomacoustics` but  I can double check. @iranroman might have some input on this as well

Reply:
I think for azimuth what we want is to check the range of values, not a specific limit. So for example, in the case of degrees, the difference between min and max value should not exceed 360. Similar story for radias, but the limit is 2*pi. 

For elevation it would be 180 degrees and pi radians. 

This will allow for different conventions across different datasets, while checking for the limits that exist across all conventions.